File: speed-test/LeibnizFormula.f90
program LeibnizFormula
  implicit none
  integer :: i = 0
  real(8) :: s = 0.0
  do i=0, 10**8
     s = s + (-1.0)**i /(2.0*i + 1.0)
  end do
  print *, 'Ans:', s*4
end program LeibnizFormula
      

File: speed-test/LeibnizFormula.java
public class LeibnizFormula {
    public static void main(void){
      int i;
      double s=0d;
      for(i=0; i<=10e8; i++){
        s += Math.pow(-1.0, i)/(2.0*i+1.0);
      }
      System.out.println("Ans:" + 4*s);
    }
}
